version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.4.0-SNAPSHOT"
```

Comment:
Can we use `https://github.com/dwijnand/sbt-dynver` instead of comitting the version number to a file? I have not tried `sbt-dynver` with `sbt-release` but it might also get rid of the following release process steps: `setReleaseVersion` `commitReleaseVersion` `setNextVersion` `commitNextVersion`.